<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserConfig</class>
 <widget class="QDialog" name="UserConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>469</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>525</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configurações do usuário</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButtonSalvar">
     <property name="text">
      <string>Salvar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxEmailCompra">
     <property name="title">
      <string>E-mail Compra</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditEmail"/>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="lineEditEmailSenha">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditServidorSMTP"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditPortaSMTP"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Senha:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Servidor SMTP:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>E-mail:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Porta:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Cópia p/:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEditEmailCopia"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxAcbr">
     <property name="title">
      <string>ACBr</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Loja:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="ItemBox" name="itemBoxLoja"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Pasta PDF/Excel</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditOrcamentosFolder"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonVendasFolder">
        <property name="text">
         <string>Navegar...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Vendas:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Orçamentos:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonOrcamentosFolder">
        <property name="text">
         <string>Navegar...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditVendasFolder"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelCompras">
        <property name="text">
         <string>Compras:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditComprasFolder"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButtonComprasFolder">
        <property name="text">
         <string>Navegar...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelEntregas">
        <property name="text">
         <string>Entregas XML:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEditEntregasXmlFolder"/>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pushButtonEntregasXmlFolder">
        <property name="text">
         <string>Navegar...</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelEntregas_2">
        <property name="text">
         <string>Entregas PDF:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditEntregasPdfFolder"/>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="pushButtonEntregasPdfFolder">
        <property name="text">
         <string>Navegar...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPushButton" name="pushButtonAlterarDados">
     <property name="text">
      <string>Alterar dados do usuário</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemBox</class>
   <extends>QLineEdit</extends>
   <header>src/itembox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditServidorSMTP</tabstop>
  <tabstop>lineEditPortaSMTP</tabstop>
  <tabstop>lineEditEmail</tabstop>
  <tabstop>lineEditEmailSenha</tabstop>
  <tabstop>lineEditEmailCopia</tabstop>
  <tabstop>lineEditOrcamentosFolder</tabstop>
  <tabstop>pushButtonOrcamentosFolder</tabstop>
  <tabstop>lineEditVendasFolder</tabstop>
  <tabstop>pushButtonVendasFolder</tabstop>
  <tabstop>pushButtonAlterarDados</tabstop>
  <tabstop>pushButtonSalvar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
